alchemy/test/cloudflare/bundle-handler.ts

```diff
@@ -1,6 +1,7 @@
 import { initLogger } from "braintrust";
 // biome-ignore lint/style/useNodejsImportProtocol: we are testing `crypto` and `node:crypto`
 import crypto from "crypto";
+import debug from "debug";
```

Comment:
This test was failing if we did `import { debug }` instead of `import debug`. Wrangler exhibited the same behavior.

The Grammy test passes regardless.